src/app/components/server/StatusPill.tsx

```diff
@@ -24,71 +24,97 @@ export const StatusPillTypeMap: Record<string, StatusPillType> = {
   Fixed: "fixed",
 };
 
-type DirectTypeProps = { type: StatusPillType };
-type ExposureProps = { exposure: Exposure };
-export type Props = (DirectTypeProps | ExposureProps) & {
+export type Props = {
+  directType?: StatusPillType;
   enabledFeatureFlags?: FeatureFlagName[];
   note?: string;
+  exposure?: Exposure;
+  isRemovalUnderMaintenance?: boolean;
```

Comment:
Proposal: Instead of changing and adding specific logic to the `StatusPill` component for items under removal: Could we not just leave the component unchanged and pass the type `removal_under_maintenance` for those exposures?